From febb7e82d945593a2d42019cd3ddc50c59c47469 Mon Sep 17 00:00:00 2001
From: jason972000 <jason972000@gmail.com>
Date: Wed, 21 Feb 2018 11:54:43 -0600
Subject: [PATCH 3/4] Disable OTA for U.S. Cellular since there is no need for
 it

Branch lineage-15.1 auto-dials *22899 every reboot if this is
not disabled.

Change-Id: Id3c0fc5dcfea44dd121e25c226d235604294d130
(cherry picked from commit 6db52ecd6e877f8d919ea17610529e17a414cfbb)
---
 assets/carrier_config_311580.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/assets/carrier_config_311580.xml b/assets/carrier_config_311580.xml
index b0eefe2..7a972ce 100644
--- a/assets/carrier_config_311580.xml
+++ b/assets/carrier_config_311580.xml
@@ -2,7 +2,7 @@
 <carrier_config_list>
     <carrier_config>
         <boolean name="dtmf_type_enabled_bool" value="true" />
-        <boolean name="use_otasp_for_provisioning_bool" value="true" />
+        <boolean name="use_otasp_for_provisioning_bool" value="false" />
         <boolean name="enabledTransID" value="true" />
         <boolean name="allowAttachAudio" value="true" />
         <int name="recipientLimit" value="20" />
-- 
2.17.1

